sklearn/ensemble/_voting.py

```diff
@@ -90,7 +90,28 @@ def fit(self, X, y, sample_weight=None):
             self.named_estimators_[name] = current_est
 
         return self
-
+                 
+    def fit_transform(self, X, y=None, **fit_params):
+        """ Return class labels or probabilities for X for each estimator.
+                       
+        Return predictions for X for each estimator.
+                   
+        Parameters
+        ----------
+        X : {array-like, sparse matrix, dataframe} of shape \
+                (n_samples, n_features)
+            Input samples.
+        y : ndarray of shape (n_samples,), default=None
+            Target values (None for unsupervised transformations).
+        **fit_params : dict
+            Additional fit parameters.
+        Returns
+        -------
+        X_new : ndarray array of shape (n_samples, n_features_new)
+            Transformed array.
+        """
+        super().fit_transform(X, y=None, **fit_params)
+            
```

Comment:
```suggestion

```

Reply:
hey, @alfaro96 thanks again. I edited all changes you suggested. have a look.
really happy for your help.